test/e2e/.mocharc.js

```diff
@@ -0,0 +1,22 @@
+/**
+ * @file Top-level mocha configuration file.
+ * @author Edwin Takahashi
+ */
+
+const framework = process.env.FRAMEWORK;
+let configFile;
+
+// Sets the config variable to the appropriate configuration file
+// for the specified framework.
+// As Selenium tests are the default and in the interest of not interrupting
+// existing workflow, if the value is undefined it will default to Selenium.
+if ( framework === undefined || framework.toLowerCase() === 'selenium' ) {
+	configFile = 'lib/mocha-hooks/mocha-hooks.js';
+} else if ( framework.toLowerCase() === 'playwright' ) {
```

Comment:
Nit: it might make sense to do this instead:

```
const configFile = framework.toLowerCase() === 'playwright' ? //playwright : //selenium
```

Reason being that this would still default to selenium in the case that framework is set to something we don't support